docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -286,18 +261,34 @@ See <<IERC20-allowance,IERC20::allowance>>.
 
 See <<IERC20-transfer,IERC20::transfer>>.
 
+Requirements:
+
+- `recipient` cannot be the zero address.
+- The caller must have a balance of at least `amount`.
+
 [.contract-item]
 [[ERC20-transfer_from]]
 ==== `[.contract-item-name]#++transfer_from++#++(ref self: ContractState, sender: ContractAddress, recipient: ContractAddress, amount: u256) → bool++` [.item-kind]#external#
 
 See <<IERC20-transfer_from,IERC20::transfer_from>>.
 
+Requirements:
+
+- `sender` cannot be the zero address.
+- `recipient` cannot be the zero address.
+- `sender` must have a balance of at least `amount`.
+- The caller must have allowance for ``sender``'s tokens of at least `amount`.
+
```

Comment:
```suggestion
- `sender` cannot be the zero address.
- `sender` must have a balance of at least `amount`.
- `recipient` cannot be the zero address.
- The caller must have allowance for ``sender``'s tokens of at least `amount`.

```
what do you think about this? it groups together all reqs for `sender`

Reply:
Yeah, it's visually better. +1